internal/bminventory/inventory.go

```diff
@@ -2057,6 +2069,56 @@ func optionalParam(data *string, field string, updates map[string]interface{}) {
 	}
 }
 
+func (b *bareMetalInventory) reportUsage(enabled bool, name string, props *map[string]interface{}, usages map[string]models.Usage) {
+	if enabled {
+		b.usageApi.AddUsage(usages, name, props)
+	} else {
+		b.usageApi.RemoveUsage(usages, name)
+	}
+}
+
+func (b *bareMetalInventory) reportProxyUsage(params *models.ClusterUpdateParams, usages map[string]models.Usage) {
+	props := map[string]interface{}{}
+	enabled := false
```

Comment:
if it's nil it's doesn't mean that the feature is disabled, disabled is when the values are set to empty string, nill just mean that user didn't set anything in this update call.
I'm not sure how UI is working with the client, i mean if it's patch or put, so you need to make sure how it's used.

Reply:
fixed